postcss.config.js

```diff
@@ -46,6 +47,7 @@ const plugins = [
 		{
 			files: [
 				`${ pkg.config.coreThemeDir }/assets/pcss/custom-selectors/_variables.pcss`,
+				`${ pkg.config.coreThemeDir }/assets/pcss/icons/_variables.pcss`,
```

Comment:
Needed for `svg-load()` and `svg-inline()` functions to work in other entry points